staging/publishing/rules.yaml

```diff
@@ -1644,4 +1644,10 @@ rules:
       branch: release-1.19
       dir: staging/src/k8s.io/controller-manager
     name: release-1.19
-    go: 1.15
+- destination: mount-utils
```

Comment:
nit: add a newline to match the other entries

Reply:
done